operator/v1/types_ingress.go

```diff
@@ -1649,6 +1649,15 @@ type IngressControllerTuningOptions struct {
 	// +optional
 	TunnelTimeout *metav1.Duration `json:"tunnelTimeout,omitempty"`
 
+	// ConnectTimeout defines the maximum time to wait for
+	// a connection attempt to a server/backend to succeed.
+	//
+	// If unset, the default timeout is 5s.
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:validation:Format=duration
+	// +optional
+	ConnectTimeout *metav1.Duration `json:"connectTimeout,omitempty"`
```

Comment:
missing format test for the new field?

Reply:
I'm not sure which format test you are referring to.

Reply:
> I'm not sure which format test you are referring to.

Yeah, it's missing. Please create a test showing that an incorrect duration format is rejected.

Reply:
A validation pattern is added. The regexp is taken from other duration fields of the API.

Reply:
Please add a test for the validation on this field to https://github.com/openshift/api/blob/master/operator/v1/tests/ingresscontrollers.operator.openshift.io/AAA_ungated.yaml

Reply:
Done.